file_service/management/commands/import_files.py

```diff
@@ -0,0 +1,146 @@
+        self.bucket = self.connection.get_bucket(settings.AWS_EXPORT_BUCKET_SLIDE_TOOL, validate=False)
+        self.canvas_progress_urls = []
+
+    def handle(self, *args, **options):
+        keyword = options.get('keyword')
+        canvas_course_id = options.get('canvas_course_id')
+        csv_path = options.get('csv_path')
+
+        if csv_path:
+            self._import_csv(csv_path)
+        elif keyword and canvas_course_id:
+            self._import_isite(keyword, canvas_course_id)
+        else:
+            raise CommandError(
+                'You must provide either the --keyword and --canvas_course_id options or the --csv option.'
+            )
+
+        import_count = len(self.canvas_progress_urls)
+        failed_count = 0
+        while self.canvas_progress_urls:
+            time.sleep(2)
+            completed_imports = []
+            failed_imports = []
+            for progress_url in self.canvas_progress_urls:
+                progress = json.loads(SDK_CONTEXT.session.request('GET', progress_url).text)
+                workflow_state = progress['workflow_state']
+                if workflow_state == 'completed':
+                    completed_imports.append(progress_url)
+                elif workflow_state == 'failed':
+                    failed_imports.append(progress_url)
+                    failed_count += 1
+
+            for progress_url in completed_imports + failed_imports:
+                self.canvas_progress_urls.remove(progress_url)
+            count_processing = len(self.canvas_progress_urls)
+            if count_processing:
+                logger.info(
+                    "%d Canvas imports complete, %d failed, %d processing",
+                    len(completed_imports),
+                    len(failed_imports),
+                    count_processing
+                )
+
+        logger.info("Completed import of %d iSites file exports, %d failed.", import_count, failed_count)
+
+    def _import_csv(self, csv_path):
+        logger.info("Importing iSites file exports from csv %s", csv_path)
+        try:
+            with open(csv_path, 'rb') as csv_file:
+                for row in csv.reader(csv_file):
+                    self._import_isite(row[0], row[1])
+        except (IOError, IndexError):
+            raise CommandError("Failed to read csv file %s", csv_path)
+
+    def _import_isite(self, keyword, canvas_course_id):
+        export_file_url = self._get_export_s3_url(keyword)
+        logger.info("Importing iSites file export from %s to Canvas course %s", export_file_url, canvas_course_id)
+        response = json.loads(content_migrations.create_content_migration_courses(
+            SDK_CONTEXT,
+            canvas_course_id,
+            'zip_file_importer',
+            settings_file_url=self._get_export_s3_url(keyword),
+            settings_folder_id=self._get_root_folder_for_canvas_course(canvas_course_id)['id']
+        ).text)
```

Comment:
`settings_folder_id` is a required parameter for the content migration API endpoint. https://canvas.instructure.com/doc/api/content_migrations.html#method.content_migrations.create